frontend/src/components/Login.js

```diff
@@ -13,41 +13,41 @@ import {
 
 
 export default class Login extends Component {
-
     static defaultProps = {
-        nightMode: false,
```

Comment:
Good deletion: We'll think of designs such as night mode after releases. 
(Ignore this, just trying to think of review stuff to say).